DcCore/DcCore/Extensions/UIColor+Extensions.swift

```diff
@@ -0,0 +1,64 @@
+import UIKit
+
+internal extension UIColor {
```

Comment:
What's the reason of having this here? Don't see where we are using it.

If we want to have `UIColor+Extensions` in `DcCore` for later use, we should (try to) declare these methods in here public (and stop declaring the whole thing `internal`)so we can get rid of `deltachat-ios/Extensions/UIColor+Extensions.swift`.

Reply:
DcChat uses the extension

Reply:
I've declared the methods and the extension public and reduced the code duplication now